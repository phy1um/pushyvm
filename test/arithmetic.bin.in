10
17
9

