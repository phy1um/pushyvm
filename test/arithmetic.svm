;; test addition +5
INT[1]
lii[5]
add
pop[1]

;; test subtraction (-5)
INT[1]
lii[5]
sub
pop[1]

;; test multiplication (*3)
INT[1]
lii[3]
mul
pop[1]

INT[0]
