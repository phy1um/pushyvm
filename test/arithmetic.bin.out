15
12
27

